Tests/PubNubTests/Networking/Routers/PresenceRouterTests.swift

```diff
@@ -29,38 +29,42 @@
 import XCTest
 
 // swiftlint:disable:next type_body_length
-final class PresenceEndpointTests: XCTestCase {
+final class PresenceRouterTests: XCTestCase {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [SwiftLint rule 'type_body_length' did not trigger a violation in the disabled region. Please remove the disable command.](https://app.codacy.com/app/PubNub/swift/pullRequest?prid=4513555)